src/components/SelectorInput/SelectorInput.js

```diff
@@ -295,16 +389,23 @@ export default class SelectorInput extends React.Component<Props, State> {
         || (name && name.toUpperCase().includes(query.toUpperCase())))
       : options;
 
+    const filteredHorizontalListData = (query && query.length >= MIN_QUERY_LENGTH && horizontalOptions.length)
```

Comment:
You're becoming master of complex and unreadable code :D

Reply:
```
let filteredHorizontalListData = horizontalOptions;
if (query && query.length >= MIN_QUERY_LENGTH && horizontalOptions.length) {
  filteredHorizontalListData = horizontalOptions.filter(({ value: val, name }) => {
    val = val ? val.toUpperCase() : '';
    name = name ? name.toUpperCase() : '';
    const searchQuery = query.toUpperCase();
    return val.includes(searchQuery) || name.includes(searchQuery);
  });
}
```

Reply:
Thanks, improved in new commit 👍